content/blog/performant-code/index.Rmd

```diff
@@ -0,0 +1,530 @@
+---
+output: hugodown::hugo_document
+
+slug: performant-packages
+title: Writing Performant Code with Tidy Tools
```

Comment:
Not feeling sure how to title this one--would something like "10 Tips for..." cue more clearly the format of this post? Or something else?

Reply:
BTW the title should be sentence case

Reply:
[a6ca771](https://github.com/tidyverse/tidyverse.org/pull/633/commits/a6ca7716d5da34aa3a5aaca3fc445136901e4633), thanks!